css/styles.css

```diff
@@ -273,11 +276,14 @@ section {
 .feature {
   flex-basis: calc((100% - 48px) / 3);
 }
+.feature img {
+  width: 100%;
+}
 img {

```

Comment:
it's not a good pattern to add styles to global tag, it's better to use classes etc